hw/xbox/lpc47m157.c

```diff
@@ -47,7 +47,12 @@
 #define CONFIG_DEVICE_BASE_ADDRESS_LOW      0x61
 #define CONFIG_DEVICE_INETRRUPT             0x70
 
-#define DEBUG_LPC47M157
+// #define DEBUG
+#ifdef DEBUG
+# define XBOX_DPRINTF(format, ...)     printf(format, ## __VA_ARGS__)
+#else
+# define XBOX_DPRINTF(format, ...)     do { } while (0)
```

Comment:
Do we really need the `XBOX_` prefix?

Reply:
No, this was a copy-paste from another file, we can remove prefix